mne/channels/montage.py

```diff
@@ -206,6 +206,16 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         ch_names_ = []
         pos = []
         with open(fname) as fid:
+            # Default units are meters
+            scale_factor = 1.
+            for line in fid:
+                if 'UnitPosition' in line:
```

Comment:
I agree.  All of the `.elc` files in `channels/data/montages` do have this line, and it is always before positions.  I'm not familiar with any official specs on the file format though.